ArrayTransformation/transformation.js

```diff
@@ -0,0 +1,23 @@
+
+const plusI =(x,i)=>{
+    return x+i;
+}
+
+
+var map = function(arr, fn) {
+    var transformedArr=[]
+    for(let i=0;i<arr.length;i++){
+        transformedArr.push(fn(arr[i],i));
+    }
+    return transformedArr;
```

Comment:
Great Job, try to solve it using array built-in method. it would be more clean approch.

Reply:
Yes with built-in method map  code will be more cleaner.But in leetcode task it was mentioned that solve the task without using buit-in method.
So should I change it with built-in method?

Reply:
leetcode was just a medium for providing questions, we are looking for optimised and clean solutions here. so it would be great if we use map here. update the code accordingly.